src/attributes.md

```diff
@@ -376,6 +373,84 @@ pub mod m3 {
 }
 ```
 
+#### Must Use Attribute
+
+The `must_use` attribute can be used on user-defined composite types
+([`struct`s][struct], [`enum`s][enum], and [`union`s][union]) and [functions].
+
+When used on user-defined composite types, if the [expression] of an
+[expression statement] has that type, then the `unused_must_use` lint is
+violated.
+
+```rust
+#[must_use]
+struct MustUse {
+  // some fields
+}
+
+# impl MustUse {
+#   fn new() -> MustUse { MustUse {} }
+# }
+
```

Comment:
Add a # to the start of one of these blank lines to avoid two in the rendered text.